File: demos/java/sqs-send/src/main/java/com/mahendra/SendMessage.java
package com.mahendra;

import software.amazon.awssdk.services.sqs.SqsClient;
import software.amazon.awssdk.services.sqs.model.SendMessageRequest;
import software.amazon.awssdk.services.sqs.model.SendMessageResponse;


public class SendMessage {
    public static void main(String[] args) {
        String queueUrl = "https://sqs.us-east-1.amazonaws.com/890756660068/que1"; // Replace with your queue URL

        SqsClient sqsClient = SqsClient.builder().build();

        SendMessageRequest sendMsgRequest = SendMessageRequest.builder()
            .queueUrl(queueUrl)
            .messageBody("Hello from Java SQS client!")
            .build();

        SendMessageResponse response = sqsClient.sendMessage(sendMsgRequest);

        System.out.println("Message sent with ID: " + response.messageId());

        sqsClient.close();
    }
}

File: demos/python/04-sqs/send-message.py
import boto3

# Create SQS client
sqs = boto3.client('sqs', region_name='us-east-1')

# Replace with your SQS queue URL
queue_url = 'https://sqs.us-east-1.amazonaws.com/890756660068/que1'

# Send message to SQS queue
response = sqs.send_message(
    QueueUrl=queue_url,
    MessageBody='Hello from Python!'
)

print('Message ID:', response['MessageId'])